static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
It makes sense, it's just that I have a difficult time reconciling this with the SoC between HTML and CSS. Obviously I have a huge mental block here :blush:

I imagine myself coming across a snippet with inline styles, my first action would be to move it to a CSS file; we're so used to switch back and forth between the two when developing/debugging anyway, it's literally why CSS was invented.